kratos/tests/cpp_tests/geometries/test_projection_nurbs_geometry.cpp

```diff
@@ -188,20 +188,20 @@ typedef Node<3> NodeType;
         KRATOS_CHECK_NEAR(parameter[1], 0.5, TOLERANCE);
         KRATOS_CHECK_VECTOR_NEAR(projected_point, projected_point_expected1, TOLERANCE);
 
-        // Try projection of point (0.06, -0.06, 0.06) close to the singularity
+        // Try projection of point (0.1, -0.005, 0.005) close to the singularity
```

Comment:
you sure you didn't but a `0` too much?

Reply:
yes, I just moved the point closer but not on the singularity